assets/stylesheets/selectbox.sass

```diff
@@ -35,15 +36,16 @@
     align-items: center
     justify-content: space-between
     &:after
-      @include size(0,0)
+      width: 0
+      height: 0
       content: ''
       border-left: 5px solid transparent
       border-right: 5px solid transparent
-      border-top: 5px solid $primary-font-color
+      border-top: 5px solid black
       margin-left: 10px
   ul
     visibility: hidden
-    margin: 0
+    margin: 0 0 -1px 0
```

Comment:
should be 
`margin: -1px 0 0 0`
and for class .direction-reverse
`margin: 0 0 -1px 0`